python_modules/dagster/dagster/components/core/defs_module.py

```diff
@@ -151,9 +151,13 @@ def iterate_components(self) -> Iterator[Component]:
             yield component
 
 
-def _crawl(context: ComponentLoadContext) -> Mapping[Path, Component]:
```

Comment:
renamed this function since I'm now using it within airlift. I also made it possible to exclude certain dirs (sometimes pycache and venv files get picked up, we should probably move this stuff into here automatically)